assets/dev/js/editor/components/dynamic-tags/control-behavior.js

```diff
@@ -95,6 +95,9 @@ module.exports = Marionette.Behavior.extend( {
 			const proTeaser = Marionette.Renderer.render( '#tmpl-elementor-dynamic-tags-promo' );
 
 			$tagsListInner.append( proTeaser );
+		} else if ( elementor.helpers.hasPro() && elementorPro.config.isActive && ! Object.keys( tags ).length ) {
```

Comment:
Extract `hasPro` and `hasTags` to external variables

Reply:
Done.